README.md

```diff
@@ -46,3 +46,16 @@ https://share.github.io/sharedb/
 ### Leaderboard
 
 [<img src="examples/leaderboard/demo.gif" height="436">](examples/leaderboard)
+
+## Development
+
+### Documentation
+
+The documentation is stored as Markdown files, but sometimes it can be useful to run these locally. The docs are served using [Jekyll](https://jekyllrb.com/), and require Ruby and [Bundler](https://bundler.io/).
```

Comment:
```suggestion
The documentation is stored as Markdown files, but sometimes it can be useful to run these locally. The docs are served using [Jekyll](https://jekyllrb.com/), which requires Ruby > 2.4.0 and [Bundler](https://bundler.io/).
```